scripts-available/CDB_Quota.sql

```diff
@@ -26,19 +26,14 @@ BEGIN
       WHERE o_table_schema = schema_name AND o_table_catalog = current_database()
   ),
   user_tables AS (
-    SELECT table_name FROM information_schema.tables
-      WHERE table_catalog = current_database() AND table_schema = schema_name
-        AND table_name != 'spatial_ref_sys'
-        AND table_name != 'cdb_tablemetadata'
-        AND table_type = 'BASE TABLE'
+    SELECT table_name FROM _CDB_NonAnalysisTablesInSchema(schema_name)
```

Comment:
For the reference, here's the function's definition: https://github.com/CartoDB/cartodb-postgresql/pull/280/files#diff-00e1865d7890cd4d9649d8c8580cd043R29